test/tests/functional/pbs_svr_dyn_res.py

```diff
@@ -588,3 +607,122 @@ def test_res_boolean_invalid_input(self):
+
+        scr_body = ['echo "10"', 'exit 0']
+        home_dir = os.path.expanduser("~")
+        fp = self.du.create_temp_file(body=scr_body, dirname=home_dir)
+        # Add to filenames for cleanup
+        self.filenames.append(fp)
+
+        dyn_scr = '"foo !' + fp + '"'
+        self.scheduler.set_sched_config({'server_dyn_res': dyn_scr},
+                                        validate=False)
+
+        # give write permission to group and others
+        self.du.chmod(path=fp, mode=0766)
+        self.check_access_log(fp)
+
+        # give write permission to group
+        self.du.chmod(path=fp, mode=0764)
+        self.check_access_log(fp)
+
+        # give write permission to others
+        self.du.chmod(path=fp, mode=0746)
+        self.check_access_log(fp)
+
+        # give write permission to user only
+        self.du.chmod(path=fp, mode=0744)
+        self.check_access_log(fp, exist=False)
+
+        # Create a script in home directory which has more open privileges
+        # This should make loading of this file fail in all cases
+        dir_temp = self.du.mkdtemp(mode=0766, dir=home_dir)
+        fp = self.du.create_temp_file(body=scr_body, dirname=dir_temp)
+        # Add to filenames for cleanup
+        self.filenames.append(fp)
+        self.filenames.append(dir_temp)
+
+        dyn_scr = '"foo !'+fp+'"'
+        self.scheduler.set_sched_config({'server_dyn_res': dyn_scr},
+                                        validate=False)
+
+        # give write permission to group and others
+        self.du.chmod(path=fp, mode=0766)
+        self.check_access_log(fp)
+
+        # give write permission to group
+        self.du.chmod(path=fp, mode=0764)
+        self.check_access_log(fp)
+
+        # give write permission to others
+        self.du.chmod(path=fp, mode=0746)
+        self.check_access_log(fp)
+
+        # give write permission to user only
+        self.du.chmod(path=fp, mode=0744)
+        self.check_access_log(fp)
+
+        # Create a dynamic script with right permissions
+        fp = self.du.create_temp_file(body=scr_body, dirname=home_dir)
+        # Add to filenames for cleanup
+        self.filenames.append(fp)
+
+        dyn_scr = '"foo !' + fp + '"'
+        # give write permission to user only
+        self.du.chmod(path=fp, mode=0744)
+        self.scheduler.set_sched_config({'server_dyn_res': dyn_scr},
+                                        validate=False)
+        self.check_access_log(fp, exist=False)
+
+        time.sleep(1)
+        match_from = int(time.time())
+        # give write permission to others
+        self.du.chmod(path=fp, mode=0746)
+        self.server.manager(MGR_CMD_SET, SERVER, {'scheduling': 'True'})
+        self.scheduler.log_match(fp + ' file has a non-secure file access',
+                                 starttime=match_from, existence=True,
+                                 max_attempts=10)
+
+        # Create dynamic resource script in tmp directory and check
```

Comment:
Same comment here, can you please explain what additional testing this section of the test will perform?